roles/azimuth/defaults/main.yml

```diff
@@ -104,16 +104,22 @@ azimuth_curated_sizes: []
 #   Configuration for the password authenticator
 azimuth_authenticator_password_enabled: true
 azimuth_authenticator_password_label: "Username + Password"
+azimuth_authenticator_password_hidden: false
 azimuth_authenticator_password_auth_url: "{{ azimuth_openstack_auth_url }}"
 azimuth_authenticator_password_domain: "{{ azimuth_openstack_domain }}"
 azimuth_authenticator_password_verify_ssl: "{{ azimuth_openstack_verify_ssl }}"
 #   Configuration for the appcred authenticator
-azimuth_authenticator_appcred_enabled: false
+#     By default, the appcred authenticator is enabled but hidden
+#     This is useful when using the SDK as automation can be given an appcred
+#     but the option is not presented to users in the choice form
+azimuth_authenticator_appcred_enabled: true
 azimuth_authenticator_appcred_label: "Application Credential"
+azimuth_authenticator_appcred_hidden: true
 azimuth_authenticator_appcred_auth_url: "{{ azimuth_openstack_auth_url }}"
 azimuth_authenticator_appcred_verify_ssl: "{{ azimuth_openstack_verify_ssl }}"
 #   Configuration for the federated authenticator
 azimuth_authenticator_federated_enabled: false
+azimuth_authenticator_federated_hidden: false
```

Comment:
Why?

Reply:
Because we want the federated options to be visible to users when they are enabled right?

Reply:
The hidden option was added so we can enable the appcred authenticator always, but not have it visible to users (by default).